spring-cloud-bus/src/main/java/org/springframework/cloud/bus/jackson/BusJacksonAutoConfiguration.java

```diff
@@ -39,23 +39,20 @@
 public class BusJacksonAutoConfiguration {
 
 	@Bean
+	@ConditionalOnMissingBean(name="busJsonConverter")
```

Comment:
Haha! I haven't looked, but I may have written that :-) You're OK if that's what was already there.